### ISSUE TITLE
[Adaptive Cards - Sample - Inputs with validation] - After applying text spacing, 'I accept the terms and conditions (True/False)' checkbox control text is getting truncated and not properly visible on the 'Inputs with validation sample' page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9308]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Tool: Text Spacing (Stylus)

**Repro Steps:**

1. Apply Text Spacing tool on the page.
2. Open the above URL on the Edge Dev browser.
3. Navigate to the 'Samples' control present on the header section of the page using tab key and activate using enter key.
4. Navigate to the 'Inputs with validation' control present on the left navigation using tab key and activate using enter key.
5. Navigate to the 'I accept the terms and conditions (True/False)' checkbox control using tab key.
6. Verify that, after applying text spacing, 'I accept the terms and conditions (True/False)' checkbox control text is properly visible on the 'Inputs with validation sample' page or not.

**Actual Result:**
After applying text spacing, 'I accept the terms and conditions (True/False)' checkbox control text is getting truncated and not properly visible on the 'Inputs with validation sample' page.

**Observation:**
Tooltip did not appear on hovering with the mouse.

**Expected Result:**
After applying text spacing, 'I accept the terms and conditions (True/False)' checkbox control text should be properly visible without any cropping/overlapping on the 'Inputs with validation sample' page.

**User Experience:**
Low vision persons who increase spaces between characters, words, etc. to read content will be impacted and might not be able to see the 'I accept the terms and conditions (True/False)' checkbox control text properly if after applying text spacing it is getting truncated.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/f408c9cf-cf3b-428a-9e7a-9129132f6fab)

### Card JSON

```JSON
NA
```
